respokeSDK/build.gradle

```diff
@@ -17,12 +17,16 @@ repositories {
 }
 
 android {
-    compileSdkVersion 22
-    buildToolsVersion '22.0.1'
+    compileSdkVersion 23
+    buildToolsVersion '23.0.3'
+
+    configurations {
+        javadocDeps
+    }
 
     defaultConfig {
         minSdkVersion 15
-        targetSdkVersion 22
+        targetSdkVersion 23
```

Comment:
22 still sits at 23.2% of devices [1]. This seems a bit aggressive but I could be wrong.

[1] https://developer.android.com/about/dashboards/index.html